app/scripts/models/resume-token.js

```diff
@@ -19,7 +19,9 @@ define([
       campaign: undefined,
       entrypoint: undefined,
       state: undefined,
-      verificationRedirect: undefined
+      verificationRedirect: undefined,
+      // fields from a User
+      uniqueUserId: undefined
```

Comment:
can uniqueUserId be before verificationedirect? I think it was in alphabetical order before.
@shane-tomlinson